website/content/docs/plugins/creation/index.mdx

```diff
@@ -272,23 +257,26 @@ The documentation structure needed for Packer.io can be generated manually, by c
 [[ -d docs/ ]] && zip -r docs.zip docs/
 ```
 
+</Tab>
+</Tabs>
+
 Once the first `docs.zip` file has been included into a release you will need to open a one time pull-request against [hashicorp/packer](https://github.com/hashicorp/packer) to register the plugin docs.
 
-This is done by adding the block below for the respective plugin to the file [website/data/docs-remote-navigation.js](https://github.com/hashicorp/packer/blob/master/website/data/docs-remote-plugins.json).
+This is done by adding the block below for the respective plugin to the file [website/data/docs-remote-plugins.json](https://github.com/hashicorp/packer/blob/master/website/data/docs-remote-plugins.json).
 
 ```json
 {
-   "title": "Scaffolding",
-   "path": "scaffolding",
-   "repo": "hashicorp/packer-plugin-scaffolding",
-   "version": "latest"
- }
+  "title": "Scaffolding",
+  "path": "scaffolding",
+  "repo": "hashicorp/packer-plugin-scaffolding",
+  "version": "latest",
+  "sourceBranch": "main"
```

Comment:
Even though it's optional, I've included `sourceBranch` in the config shown here for visibility. I've noted that `sourceBranch` will default to `main` in a comment below.

Reply:
I'm updating the scaffolding with the same docs 
https://github.com/hashicorp/packer-plugin-scaffolding/pull/32 👍🏼